src/ducks/Signals/signalFormManager/signalMaster/SignalMaster.js

```diff
@@ -143,8 +146,12 @@ const SignalMaster = ({
         <SharedTriggerForm
           id={stateTrigger.id}
           trigger={stateTrigger}
-          onOpen={setOpenSharedForm}
-          onCreate={() => handleSettingsChange(settings)}
+          onOpen={data =>
+            isLoggedIn ? setOpenSharedForm(data) : setAnonWarning(true)
```

Comment:
setAnonWarning can have default value. Did you pass false in other places?
From naming of this func it should works without arguments. @GarageInc

Reply:
It is good point. Child should't know about props of parent.